alacritty_terminal/src/grid/resize.rs

```diff
@@ -355,7 +355,7 @@ impl<T: GridCell + Default + PartialEq + Clone> Grid<T> {
                     }
                     row = Row::from_vec(wrapped, occ);
 
-                    if i <= self.display_offset {
```

Comment:
I don't think the zero check is necessary anymore with this? Since both are a usize it's impossible for i to ever be smaller than the display offset if it is zero.